thirdeye/thirdeye-frontend/app/pods/components/self-serve-alert-yaml-details/template.hbs

```diff
@@ -1,103 +1,79 @@
-<div class="te-search-results__header">
-  <div class="te-search-results__title-group">
+<div class="te-search-results__header row">
+  <div class=" te-search-results__title-group col-md-10 col-xs-9">
```

Comment:
The indent issue before `te-search-results`

Reply:
good catch, fixed.